### PR TITLE
Feat: Added a component for truncated ids.

### PR DESCRIPTION
**Pull request by abbasabid**
# Description of change

- Added a component for truncated ids on Landing page and Address Page.
- Ids can now be find with CTRL+F.

Resolves https://github.com/iotaledger/explorer/issues/587

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by msarcev** [comment]
Please, `TruncateId` -> `TruncatedId` everywhere.

File: client/src/app/components/stardust/TruncatedId.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.truncated-id {
+    span {
+        width: 100px;
```
Comment (brancoder) [inline client/src/app/components/stardust/TruncatedId.scss:3]: Maybe we should consider not limiting the `truncated-id` to `width: 100px;` rather we could have truncated-id be width:100% and add `max-width` on the container element or something along these lines. That way the truncated-id would shrink or get larger base on the containers width. Wdyt?

**Review by brancoder — CHANGES_REQUESTED** [review]
We should aim to have all truncated ids throughout the project displayed in the same form:
1. On Transaction page we still display them with 4 dots in the middle:
image
2. On Address page all of the tabs for smaller screens display the ids with dots in the middle:
image
image

**Review by brancoder — COMMENTED** [review]
Maybe we should change the `.card` class on [this](https://github.com/iotaledger/explorer/blob/feat/findable-truncated-ids/client/src/app/components/stardust/AssociationSection.tsx#L150) line because this is a `td` in `association-section--table` and that doesn't seam correct. The `.card` class is used on the same page in section `.association-section--cards` where its appropriate. Wdyt?

**Comment by abbasabid** [comment]
> this

Fixed.

**Comment by abbasabid** [comment]
> Maybe we should change the `.card` class on [this](https://github.com/iotaledger/explorer/blob/feat/findable-truncated-ids/client/src/app/components/stardust/AssociationSection.tsx#L150) line because this is a `td` in `association-section--table` and that doesn't seam correct. The `.card` class is used on the same page in section `.association-section--cards` where its appropriate. Wdyt?

Fixed.

**Review by brancoder — APPROVED** [review]
LGTM

**Pull request merged**